src/lib.rs

```diff
@@ -43,61 +46,76 @@
 
 #![deny(warnings)]
 #![deny(missing_docs)]
+#![cfg_attr(docsrs, feature(doc_cfg))]
```

Comment:
This annotation is required to allow use of the `#[cfg_attr(docsrs, doc(cfg(feature = "async")))]` statements, which will label modules as requiring a particular feature when on docs.rs

Reply:
fixed